pkg/autodiscover/autodiscover.go

```diff
@@ -84,7 +89,10 @@ func DoAutoDiscover() (env configuration.TestParameters,
 	debugPods = findPodsByLabel(oc.Coreclient, debugLabels, debugNS)
 	crds = FindTestCrdNames(testData.CrdFilters)
 	csvs = findOperatorsByLabel(oc.OlmClient, []configuration.Label{{Name: tnfCsvTargetLabelName, Prefix: tnfLabelPrefix, Value: tnfCsvTargetLabelValue}}, testData.TargetNameSpaces)
-	return env, testData, pods, debugPods, crds, namespaces, csvs
+	dps := findDeploymentByLabel(oc.AppsClients, testData.TargetPodLabels, namespaces)
+	statefulSets := findStatefulSetByLabel(oc.AppsClients, testData.TargetPodLabels, namespaces)
+	hpas := findHpaControllers(*oc.K8sClient, namespaces)
+	return env, testData, pods, debugPods, crds, namespaces, csvs, dps, statefulSets, hpas
```

Comment:
Just a thought, instead of returning all of these values should we encapsulate these into a struct and then just return that?  That way if we need to add/remove more values in the future it's easier just to add the variable into the struct instead of modifying this list of returned vars?